src/Search/Elastica/QueryFactory.php

```diff
@@ -103,6 +103,8 @@ protected function applyDateFilters(SearchNodesRequest $request, ParsedQuery $pa
             ['query' => 'created_before', 'field' => 'created_at', 'operator' => ComparisonOperator::LT()],
             ['query' => 'updated_after', 'field' => 'updated_at', 'operator' => ComparisonOperator::GT()],
             ['query' => 'updated_before', 'field' => 'updated_at', 'operator' => ComparisonOperator::LT()],
+            ['query' => 'published_after', 'field' => 'published_at', 'operator' => ComparisonOperator::LT()],
```

Comment:
@cohencodes comparison operator should be GT for the _after_

Reply:
fixed